CHANGELOG.md

```diff
@@ -22,11 +22,11 @@
 
 🔧 Fixes:
 
-- Pull Request Title goes here
+- update template.njk to add IE meta tag
```

Comment:
Thanks for picking this up so quickly, it looks really good. I have one small note about the CHANGELOG entry:

We try to keep the template in the CHANGELOG for future people to copy and paste, so maybe we could update this to be:

```suggestion
- Pull Request Title goes here

  Description goes here (optional)

  ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))

- Improve rendering in older Internet Explorer
```

That said if you want us to do this on your behalf let me know and I'll get this merged in.

Reply:
Thanks, @nickcolley for the help. I have made the necessary changes. Please review.